showcase/vue-showcase/src/components/other-elements/OtherElements.vue

```diff
@@ -17,6 +18,8 @@ import {
     <DbProgress :value="60" :max="100" />
     <p>DbTag:</p>
     <DbTag>default</DbTag>
+    <p>DbLogo:</p>
+    <DbLogo></DbLogo>
     <DbTag variant="informative">informative</DbTag>
```

Comment:
![image](https://user-images.githubusercontent.com/104347736/217454099-1287ea97-bfca-4cb5-bce9-0c647e94bdd5.png)
Guess this is wrong copy&paste here